spec/system/ai_helper/ai_proofreading_spec.rb

```diff
@@ -0,0 +1,46 @@
+# frozen_string_literal: true
+
+RSpec.describe "AI Composer Proofreading Features", type: :system, js: true do
+  fab!(:user) { Fabricate(:admin) }
```

Comment:
```suggestion
  fab!(:current_user) { Fabricate(:admin) }
```

convention in system specs, makes things clearer